src/General/Badge/Badge.tsx

```diff
@@ -16,7 +15,6 @@ export const Badge: React.FunctionComponent<Props> = ({
       <BadgeContainer
         className={classNames('aries-badge', className)}
         role="presentation"
-        sup={sup}
```

Comment:
Hi @ritikamotwani, was wondering have you checked `sup` prop is not used anywhere? 🙏

Reply:
Yes, checked!